### PR TITLE
Allow borrowers to create 180 day term loans if credit score is at least 200

### PR DESCRIPTION
**Pull request by MiksuJak**

### CODE REVIEW

**Comment by MiksuJak** [comment]
I am really open to suggestions for variable names

File: test/truefi2/TrueLender2.test.ts [hunk 1]
```diff
@@ -310,13 +399,42 @@ describe('TrueLender2', () => {
         await expect(lender.connect(borrower).fund(loan1.address))
           .to.be.revertedWith('TrueLender: Loan risk is too high')
       })
+
+      it('loan term exceeds max term', async () => {
+        await rater.connect(borrower).submit(loan1.address)
+        await rater.yes(loan1.address)
+        await timeTravel(7 * DAY + 1)
+        await lender.setLongLoanMinTerm(DAY * 364)
+        await lender.setMaxLoanTerm(DAY * 364)
```
Comment (barrutko) [inline test/truefi2/TrueLender2.test.ts:408]: Why 364 not 365?

Comment (MiksuJak) [reply test/truefi2/TrueLender2.test.ts:408]: Because loan1 has a 1 year term and we want max term to be lower for this test

Comment (skalermo) [reply test/truefi2/TrueLender2.test.ts:408]: I think it would be more obvious if it was `DAY * 365 - 1`

File: contracts/truefi2/TrueLender2.sol [hunk 2]
```diff
@@ -257,7 +332,10 @@ contract TrueLender2 is ITrueLender2, UpgradeableClaimable {
 
         uint256 amount = loanToken.amount();
         (uint256 start, uint256 no, uint256 yes) = ratingAgency.getResults(address(loanToken));
+        uint256 term = loanToken.term();
 
+        require(term <= maxLoanTerm, "TrueLender: Loan's term is too long");
+        require(_isCredibleForTerm(term), "TrueLender: Credit score is too low for loan's term");
```
Comment (barrutko) [inline contracts/truefi2/TrueLender2.sol:338]: Let's remove underscore

File: contracts/truefi2/TrueLender2.sol [hunk 3]
```diff
@@ -257,7 +332,10 @@ contract TrueLender2 is ITrueLender2, UpgradeableClaimable {
 
         uint256 amount = loanToken.amount();
         (uint256 start, uint256 no, uint256 yes) = ratingAgency.getResults(address(loanToken));
+        uint256 term = loanToken.term();
 
+        require(term <= maxLoanTerm, "TrueLender: Loan's term is too long");
```
Comment (barrutko) [inline contracts/truefi2/TrueLender2.sol:337]: Let's wrap this with function like the rest of the requirements here.

**Review by barrutko — APPROVED** [review]

**Review by yuchenlintt — COMMENTED** [review]
The logic is straightforward and LGTM.

I think the naming is mostly okay too. I would probably choose `longTermLoanTermThreshold` to match `longTermLoanScoreThreshold` (but I'm sleepy enough now that I don't know if this name would still make sense if I were more awake).

**Pull request merged**